src/lib/y2firewall/dialogs/main.rb

```diff
@@ -39,7 +39,8 @@ def initialize
       end
 
       def title
-        _("Firewall")
+        # FIXME: the api mode is temporal, should be removed
```

Comment:
Do you need to show the `mode` here? Is it translated?

Reply:
No, we do not need it